packages/web-components/src/components/va-memorable-date/test/va-memorable-date.e2e.ts

```diff
@@ -1,6 +1,8 @@
 import { newE2EPage } from '@stencil/core/testing';
 import { axeCheck } from '../../../testing/test-helpers';
 
+const maxYear = new Date().getFullYear() + 100;
```

Comment:
This value is how we check for [max year validation here](https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/utils/date-utils.ts#L2) so I am just replicating it for this test to avoid hard coding the year value. Otherwise, tests that use that value will fail every new year.